exercicios/para-casa/Sayonara/SAYONARA.sistema_de_alunas.py

```diff
@@ -0,0 +1,152 @@
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+                
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+        return codigo_opcao
+    
+def incluir_nova_aluna():
+    print("Informe os dados ")
+    nome= input("Informe o nome e o sobrenome :")
+    turma = input("informe a turma :" )
+    notas = pedir_notas()
+    nota_participaçao=float(input("participação da aluna: "))
+    presença= chamada() 
+    dataset[(nome.split()[0] , nome.split()[1])] = {
+        "Turma": turma,
+         "Notas": notas,
+         "Presença": presença,
+         "Paticipaçao": nota_participaçao
+    }
+    print("Aluna cadastrada com sucesso ")
+    pass
+    #TODO - Implentar a função
+def pedir_notas():
+   #quantidade_notas=input("informa a quantidade das notas:")
+   notas=[]
+   for i in range(3):
+         nota=float(input(f"Digite a {i+1}° nota:" ))
+         notas.append(nota)
+
+
+   return notas 
+ 
+def chamada(): 
+    aulas=[] 
+    for  i in range (5): 
+     aula=int(input(f" Digite 1 se presente ou 0 se ausente, na aula :{i+1}°: presença:"))  
+    aulas.append(aula)
+    return aulas 
+
+    
+def consultar_lista_alunas(dataset):
+     print("Lista alunas ")
+     for aluna_tuple in dataset:
+        nome_c = aluna_tuple[ 0 ]  + " " + aluna_tuple[1]
+        nome =nome_c.strip('(').strip()
+        print (nome)
+
+     pass
+    #TODO - Implentar a função
+    
+def consultar_faltas_aluna():
+     print("informe o nome da aluna")
+     nome_completo=input("Nome completo")
+     nome_aluna,sobrenome_aluna=nome_completo.split()
+     chave_aluna=(nome_aluna,sobrenome_aluna)
+
+     if chave_aluna in dataset:
+      presencas=dataset[chave_aluna]["Presença"]
+      numero_faltas=presencas.count(False)
+      print(f" Número de faltas de {nome_aluna}  {sobrenome_aluna} :{numero_faltas}")
+     
+
+            
+        
+     return
+
+     pass
+    #TODO - Implentar a função
+    
+def consultar_notas_aluna():
+    print("Informe o nome da aluna")
+    nome_completo=input("Nome completo: ")
+    nome_aluna,sobrenome_aluna=nome_completo.split()
+    chave_aluna=(nome_aluna,sobrenome_aluna)
+    if chave_aluna in dataset:
+      nota=dataset[chave_aluna]["Notas"]
+
+
+
+
+
+      print(f" Notas de {nome_completo}:{nota}" )
+
+ 
+
+
+
+
+   
+    return
+    pass
+    #TODO - Implentar a função
```

Comment:
Podemos remover o pass, TODO e o excesso de quebra de linhas também.

**Sugestão:**
```suggestion
      print(f" Notas de {nome_completo}:{nota}" )
```